config/initializers/omniauth.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :google_oauth2, ENV['GOOGLE_CLIENT_ID'], ENV['GOOGLE_CLIENT_SECRET'], skip_jwt: true
```

Comment:
Do we need to put these into Heroku?

Reply:
Yes we will need to add them to Heroku for the release. We also will need to make sure the auth callback URL in Heroku works as well when we release

Reply:
Okay, cool. Let me know if you need help with that.